flatsurf_algorithms.tex

```diff
@@ -766,17 +766,17 @@ \subsection{The tangent space to a translation surface}
 \begin{lemma}
 Let $T$ be a combinatorial triangulation with $d$ edges. The sum of triangle
 relations is zero and it is the only relation between them. In particular,
-$V(T)$ has codimension $d-1$.
+$V(T)$ has codimension $\textrm{faces}-1$.
```

Comment:
I agree that $d$ is the number of edges. But `$\textrm{faces}$` is not so nice for the number of faces. At some point, I thought about using `n_v`, `n_e` and `n_f` for number of vertices, edges and faces.

Reply:
Ok. I'll fix it.